install.sh

```diff
@@ -51,28 +53,42 @@ spec:
     registryPoll:
       interval: 10m0s
 EOD
+  fi
+else
+  CATSRC_CANDIDATES=(
+    operatorhubio-catalog
+    community-operators
```

Comment:
we should not look into `community-operators` anymore, but instead in case of openshift we should look in `redhat-operators`

Reply:
Indeed, thx.